### PR TITLE
Epix 2 support

### PR DESCRIPTION
**Pull request by ysavary**
Hi @s-reynier,

I finally bought an Epix 2, but I discovered that this watch is not supported. It's a good reason to start with Monkey C!

### CODE REVIEW

File: manifest.xml [hunk 1]
```diff
@@ -1,29 +1,34 @@
-<!-- This is a generated file. It is highly recommended that you DO NOT edit this file. --><iq:manifest xmlns:iq="http://www.garmin.com/xml/connectiq" version="3">
+<?xml version="1.0"?>
+<!-- This is a generated file. It is highly recommended that you DO NOT edit this file. -->
```
Comment (ysavary) [inline manifest.xml:2]: As advised, I edited this file with Monkey C VSCode plugin action `Edit product` but this action re-ordered the items.

File: source/windsView.mc [hunk 2]
```diff
@@ -83,27 +84,30 @@ class windsView extends WatchUi.View {
 
 		
 	function requestWindInformationByCode(code) as Void {
-			
-		Communications.makeJsonRequest(
```
Comment (ysavary) [inline source/windsView.mc:87]: `makeJsonRequest` is deprecated: https://developer.garmin.com/connect-iq/api-docs/Toybox/Communications.html#makeJsonRequest-instance_function.

File: source/windsView.mc [hunk 3]
```diff
@@ -83,27 +84,30 @@ class windsView extends WatchUi.View {
...
-		"https://winds.mobi/api/2.2/stations/" + code,
-		{
-		},
-		{
-		"Content-Type" => Communications.REQUEST_CONTENT_TYPE_URL_ENCODED
-		},
-		method(:setStationInfo)
+		Communications.makeWebRequest(
```
Comment (ysavary) [inline source/windsView.mc:87]: makeWebRequest is available since API 1.3.0

**Comment by s-reynier** [comment]
hi @ysavary 
Garmin Epix 2 it's a very nice watch !
Have you test makeWebRequest with older watch Garmin ? like fenix 3 ?

File: manifest.xml (hunk 1 above)
Comment (s-reynier) [reply manifest.xml:2]: ok thanks

**Comment by ysavary** [comment]
> hi @ysavary Garmin Epix 2 it's a very nice watch ! Have you test makeWebRequest with older watch Garmin ? like fenix 3 ?

@s-reynier I just did it now and it's working on the simulator. I was confident because the new call `makeWebRequest` is marked "Since API 1.3.0" and the `minApiLevel` field of [manifest.xml](https://github.com/winds-mobi/winds-mobi-client-garmin/blob/master/manifest.xml#L4) is "1.4.0".

Fenix 3 is API 1.4.0.

**Pull request merged**